raising-jiras.md

```diff
@@ -0,0 +1,87 @@
+#Raising Jiras
+
+<p>Jiras are used at Holiday Extras as a means for teams to communicate what work is required (including data and insights). This doc seeks to outline some general guidelines and best practice when writing a jira.</p>
+
+##Title:
+<p>This is a brief description of the problem to be solved. Be clear and to the point.</p>
+
+**Good example:**
+<p>"Change the endpoints in HAPI to new Horizon service for Flexible Car Hire"</p>
```

Comment:
I like @robhuzzey suggestion `Allow flexible car hire to be sold via our API` would be a better title. `Change the endpoints in HAPI to new Horizon service for Flexible Car Hire` is too prescriptive and often means some people might do exactly that even if the correct solution is not to change endpoints. Also endpoints probably does not mean much to those outside the dev team and I feel all tickets should be readable by the whole business.

Maybe can have 3-5 examples of great ticket titles, and 3-5 examples of bad ticket titles. I can propose some if that would help?